### ISSUE TITLE
Grid in dialogue

### ISSUE THREAD

**Issue by jabirmoolur** [volkanceylan/Serenity#4116]
How to open a filtered grid on clicking button from form?
i.e i want to display ProductsDialogue with pre filtered filter  based on criteria (or id) value from form?

**Comment by Zahar661** [comment]
https://github.com/volkanceylan/Serene/blob/master/Serene/Serene.Web/Modules/Northwind/Order/OrderIndex.cshtml

**Comment by jabirmoolur** [comment]
@Zahar661 
Actually i want to place a grid in template dialog, not in page

**Comment by jabirmoolur** [comment]
I found way to instantiate grid in Dialog

 protected onDialogOpen() {
            $(".ui-dialog-titlebar-maximize").click();            
            super.onDialogOpen();   
            **var TrnGridd= new SHP.Default.TransactionDetailsGridProduct($('#TrnGridd'));**
            TrnGridd.PrdId = this.ProdId;
            TrnGridd.refresh();
        }    

        protected getTemplate() {
            return "<div style='min-width:800px' id='TrnGridd'></div>";
        }

This will work?
But how to pass value from Grid to Dialog, Thats synching one varibale ,
basically my dialog wants to know clicked item in Grid.

**Comment by Zahar661** [comment]
This example watched? Here and grid in the dialog and parameter transfer
![image](https://user-images.githubusercontent.com/26430136/49632876-bc472500-fa10-11e8-9d10-d1f37bfcb7de.png)
